reference_data/management/commands/update_dbnsfp_gene.py

```diff
@@ -3,8 +3,6 @@
 from reference_data.management.commands.utils.update_utils import GeneCommand, ReferenceDataHandler
 from reference_data.models import dbNSFPGene
 
-logger = logging.getLogger(__name__)
```

Comment:
By removing this the `logging ` import at the top of the file is no longer used and should be removed

Reply:
Okay